pcsx2/GS/Renderers/OpenGL/GLState.cpp

```diff
@@ -22,6 +22,9 @@ namespace GLState
 	GSVector2i viewport;
 	GSVector4i scissor;
 
+	bool point_size = false;
```

Comment:
any chance we can give this a bit better name? "point_size = yes" sounds weird :P{